net/haproxy/src/opnsense/mvc/app/views/OPNsense/HAProxy/index.volt

```diff
@@ -308,50 +361,155 @@ POSSIBILITY OF SUCH DAMAGE.
+            <br/>
+        </div>
+    </div>
+
+    <div id="subtab_haproxy-real-servers-introduction" class="tab-pane fade">
+        <div class="col-md-12">
+            <h1>Real Servers</h1>
+            <p>{{ lang._('HAProxy needs to know which servers should be used to serve content. The following minimum information must be provided for each server:') }}</p>
+            <ul>
+              <li>{{ lang._('%sFQDN or IP:%s The IP address or fully-qualified domain name that should be used when communicating with your server.') | format('<b>', '</b>') }}</li>
+              <li>{{ lang._('%sPort:%s The TCP or TCP port that should be used.') | format('<b>', '</b>') }}</li>
+            </ul>
+            <p>{{ lang._("Please note that advanced mode settings allow you to disable a certain server or to configure it as a backup server in a Backend Pool. Another neat option is the possibility to adjust a server's weight relative to other servers in the same Backend Pool.") }}</p>
+            <br/>
+        </div>
+    </div>
+
+    <div id="subtab_haproxy-virtual-services-introduction" class="tab-pane fade">
+        <div class="col-md-12">
+            <h1>Virtual Services</h1>
+            <p>{{ lang._("HAProxy requires two virtual services for it's loadbalancing and proxying features. The following virtual services must be configured for everything that should be served by HAProxy:") }}</p>
+            <ul>
+              <li>{{ lang._('%sBackend Pools:%s The HAProxy backend. Group the %spreviously added servers%s to build a server farm. All servers in a group usually deliver the same content. The Backend Pool cares for health monitoring and load distribution. A Backend Pool must also be configured if you only have a single server. The same Backend Pool may be used for multiple Public Services.') | format('<b>', '</b>', '<b>', '</b>') }}</li>
+              <li>{{ lang._('%sPublic Services:%s The HAProxy frontend. The Public Service listens for client connections, optionally applies rules and forwards client request data to the selected Backend Pool for load balancing (or proxying). Every Public Service needs to be connected to a %spreviously created Backend Pool%s.') | format('<b>', '</b>', '<b>', '</b>') }}</li>
+            </ul>
+            <p>{{ lang._('Remember to add firewall rules for all configured Public Services.') }}</p>
+            <br/>
+        </div>
+    </div>
+
+    <div id="subtab_haproxy-rules-checks-introduction" class="tab-pane fade">
+        <div class="col-md-12">
+            <h1>Rules &amp; Checks</h1>
+            <p>{{ lang._("These are all optional features, HAProxy can be used without them. After getting used to HAProxy you'll find these optional features pretty useful:") }}</p>
```

Comment:
"features, " becomes "features; " or dop the second part of the sentence, also contraction -> you will, remove "pretty"

In a cold technical manner, the sentences could simply read: "After getting acquainted with HAProxy the following optional features may prove useful."